containers/Dockerfile.build

```diff
@@ -1,30 +1,24 @@
 ARG BASE_IMAGE
 FROM ${BASE_IMAGE}
 
-ARG PACKAGE
-# Need to specify compiler package name for install and module name for build separately
-# e.g. for the latest intel this would be:
-# COMPILER_NAME=intel
-# COMPILER_PACKAGE=intel-oneapi-compilers
-ARG COMPILER_NAME
+ARG PACKAGE_NAMES="mom5 cice5 oasis3-mct libaccessom2"
```

Comment:
So *if* what I was saying above about specifying the model above is correct, here some of the dependency mapping type stuff might still be useful. 
At the very least a `yaml` file something like
```yaml
access-om2: mom5 cice5 oasis3-mct libaccessom2
```

Could be defined and then used to populated the package names here? An alternative would be to have multiple `Dockerfile.build` files named for each model, e.g. `Dockerfile.access-om2.build`

Reply:
Dependency mapping still has a home, it seems! I'll update the build.

Reply:
> Dependency mapping still has a home, it seems! I'll update the build.

Or have multiple Dockerfiles. It's not a terrible option.

Reply:
Added multiple dockerfiles :)